custom_components/alexa_media/const.py

```diff
@@ -30,6 +30,7 @@
     "sensor",
     "alarm_control_panel",
     "light",
+    "binary_sensor"
```

Comment:
I forget if the manifest.json or hacs.json may require and update for this too. Please check.

Reply:
There's no reference of the other constants if hacs.json or manifest.json, so I guess there's no need for binary_sensor too.